src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -168,11 +168,11 @@ suggestAction dflags packageExports ideOptions parsedModule text diag = concat
     , suggestFixConstructorImport text diag
     , suggestModuleTypo diag
     , suggestReplaceIdentifier text diag
-    , suggestConstraint text diag
     , removeRedundantConstraints text diag
     , suggestAddTypeAnnotationToSatisfyContraints text diag
     ] ++ concat
-    [  suggestNewDefinition ideOptions pm text diag
+    [  suggestConstraint pm text diag  
```

Comment:
Since suggestContext now uses `pm :: ParsedModule` I moved it into the list of functions that have access to `pm`. Is it OK that I changed the priority like that just to get access to it?